### PR TITLE
chore: migrate consume multiple topics example to a proper project structure

### PR DESCRIPTION
**Pull request by alxdrcirilo**
Addresses (#145).

### CODE REVIEW

**Comment by alxdrcirilo** [comment]
Hi @marcosschroh, this seems to fail with Python 3.12. People have been reporting [similar issues](https://stackoverflow.com/a/77291347).

**Review by woile — APPROVED** [review]

**Comment by woile** [comment]
Can you send a PR removing the `poetry.lock`? This library does not need it, and the latest version of aiokafka supports python3.12

File: tests/test_client.py [hunk 1]
```diff
@@ -404,16 +404,21 @@ async def test_e2e_example():
     ),
 )
 async def test_e2e_consume_multiple_topics(monitoring_enabled):
-    from examples.consume_multiple_topics import produce, stream_engine, topics
+    consume_multiple_topics_example = importlib.import_module(
```
Comment (reidmeyer) [inline tests/test_client.py:407]: oof

Comment (alxdrcirilo) [reply tests/test_client.py:407]: Indeed, but necessary if we want to keep the naming of the examples intact 😅

**Pull request merged**